package.json

```diff
@@ -6,7 +6,7 @@
   "types": "dist/src/index.d.ts",
   "scripts": {
     "test": "jest",
-    "test:e2e": "jest e2e/*  --config ./e2e/config/jest.config.json --runInBand  ",
+    "test:e2e": "jest e2e/sdk.ERC1155.e2e.test.ts  --config ./e2e/config/jest.config.json --runInBand  ",
```

Comment:
Does this mean that it will only test the `sdk.ERC1155.e2e.test.ts` file when you run the e2e?